lib/index.js

```diff
@@ -141,6 +142,7 @@ PostHTML.prototype.process = function (tree, options) {
    * @prop {?Boolean} options.skipParse - disable parsing
    */
   options = this.options = options || {}
+  this.source = tree
```

Comment:
it seems it is not being used below. is it for the plugins ?

Reply:
Yes, this is an opportunity to have access to plugins from plugins.

Reply:
plugins already have the AST right? may be this can have the source code ?

sorry I am not getting this. can you give some detail about this. as we may need to document this as well ?

Reply:
> plugins already have the AST right? may be this can have the source code ?

in this case, it is not an AST tree but the original html

> sorry I am not getting this. can you give some detail about this. as we may need to document this as well ?

Of course, for example, one of the plugins made adjustments to the AST and your plugin would like to know if there was data in the original HTML, an example invented and possibly unexplained, but this is the first thing that came to mind

Reply:
ok, I thought it that too that it was passing the HTML. just the name `tree` confused me a bit.

looks good 👍

Reply:
Yes, it confused me the same and this is an occasion for performance)